tutorials/shading/shading_reference/canvas_item_shader.rst

```diff
@@ -51,8 +51,12 @@ Global built-ins are available everywhere, including custom functions.
 +-------------------+-----------------------------------------------------------------------------+
 | Built-in          | Description                                                                 |
 +===================+=============================================================================+
-| in float **TIME** | Global time, in seconds.                                                    |
-|                   | It's subject to the rollover setting (which is 3,600 -1 hour- by default).  |
+| in float **TIME** | Global time since the shader was compiled, in seconds (always positive).    |
```

Comment:
Is this time actually reset when the shader was compiled *outside the editor*? I'm not sure.

Reply:
@clayjohn would you know about this?

Reply:
I dont think it resets when the shader is recompiled. TIME comes from ``frame.time`` which is incremented in the rasterizer.

https://github.com/godotengine/godot/blob/84e17881cd19218d32430d03613497405ad51696/drivers/gles3/rasterizer_gles3.cpp#L197-L210